src/install/kubernetes/helm/install-helm.mdx

```diff
@@ -0,0 +1,101 @@
+
+2. Use this command to add the New Relic Helm charts repo:
+    ```shell
+    helm repo add newrelic https://helm-charts.newrelic.com
+    ```
+3. Create a file named []`values-newrelic.yaml`](https://github.com/newrelic/helm-charts/blob/master/charts/nri-bundle/values.yaml), which will be used to define your configuration:
+
+    ```yaml
+    global:
+      licenseKey: _YOUR_NEW_RELIC_LICENSE_KEY_
+      cluster: _K8S_CLUSTER_NAME_
+    
+    newrelic-prometheus-agent:
+      # Automatically scrape prometheus metrics for annotated services in the cluster
+      # Collecting prometheus metrics for large clusters might impact data usage significantly
+      enabled: true
+    nri-metadata-injection:
+      # Deploy our webhook to link APM and Kubernetes entities
+      enabled: true
+    nri-kube-events:
+      # Report Kubernetes events
+      enabled: true
+    newrelic-logging:
+      # Report logs for containers running in the cluster
+      enabled: true
+    kube-state-metrics:
+      # Deploy kube-state-metrics in the cluster.
+      # Set this to true unless it is already deployed.
+      enabled: true
+    ```
+4. Make sure everything is configured properly in the chart by running the following command. Notice that we're specifying `--dry-run` and `--debug`, so nothing will be installed in this step:
+
+    ```shell
+    helm upgrade --install newrelic-bundle newrelic/nri-bundle \
+    --namespace newrelic --create-namespace \
+    -f values-newrelic.yaml \
+    --dry-run \
+    --debug
+    ```
+    
+    Please notice and adjust the following flags:
+    
+        * `global.licenseKey=YOUR_NEW_RELIC_LICENSE_KEY`: Must be set to a valid <InlinePopover type="licenseKey" /> for your account.
+        * `global.cluster=K8S_CLUSTER_NAME`: It's used to identify the cluster in the New Relic UI, so should be a descriptive value not used by any other Kubernetes cluster configured in your New Relic account.
+        * `kube-state-metrics.enabled=true`: Setting this to `true` will automatically install Kube State Metrics (KSM) for you, which is required for our integration to run. You can set this to false if KSM is already present in your cluster, even if it's on a different namespace.
+        * `newrelic-prometheus-agent.enabled=true`: Will deploy our Prometheus Agent, which automatically collects data from Prometheus endpoints present in the cluster.
+        * `nri-metadata-injection.enabled=true`: Will install our minimal webhook, which adds environment variables that, in turn, allows [linking applications instrumented with New Relic APM to Kubernetes](/docs/kubernetes-pixie/kubernetes-integration/link-your-applications/link-your-applications-kubernetes).
+
+    Our Kubernetes charts have a comprehensive set of flags and tunables that can be edited to better fit your particular needs. Please, check our [Helm chart](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle#readme) to see all the options you have.
```

Comment:
I have never heard the term "tunable"- I could be missing something and this could be a common word, and if so, sorry! But I would change this to a different word.

Reply:
I've modified the sentence to this:
`Our Kubernetes charts have a comprehensive set of flags and settings that you can edit to better meet your specific needs.`